src/NLog/Internal/FormatHelper.cs

```diff
@@ -71,9 +75,27 @@ internal static bool ObjectSupportsFormatProvider(object o)
             {
                 case TypeCode.String:   return false;
                 case TypeCode.Empty:    return false;
-                case TypeCode.Object:   return o is IFormattable;
```

Comment:
maybe we should rename `ObjectSupportsFormatProvider` now to `SkipFormattableToString`, or something like that.

Reply:
Ok. Will fix this.